pages/common/rdfind.md

```diff
@@ -0,0 +1,19 @@
+# rdfind
+
+> Finds duplicate files and gets rid of them.
```

Comment:
I think its clearer to mention - "Finds files with duplicate names". Duplicate files might mean files with the same content too.

Reply:
But it does find files of the same content, not the same name; I try to clarify that

Reply:
Ah, I understood the opposite. More the reason we should fix this. I suggest - `Find files with duplicate content ..`

Reply:
Added a pull request to resolve the issue:  https://github.com/tldr-pages/tldr/pull/2341